akka-projection-dynamodb/src/main/scala/akka/projection/dynamodb/internal/DynamoDBProjectionImpl.scala

```diff
@@ -726,6 +491,8 @@ private[projection] object DynamoDBProjectionImpl {
     import DynamoDBOffsetStore.Validation._
     implicit val ec: ExecutionContext = system.executionContext
 
+    // This is similar to DynamoDBProjectionImpl.replayIfPossible but difficult to extract common parts
+    // since this is flow processing
```

Comment:
For the flow handler I didn't see an easy way to reuse the same code so kept it as is